site/en/tutorials/structured_data/imbalanced_data.ipynb

```diff
@@ -814,7 +814,7 @@
         "    else:\n",
         "      plt.ylim([0,1])\n",
         "\n",
-        "    plt.legend()"
+        "    plt.legend();"
```

Comment:
Assuming this is for not showing text in the matplotlib output

Reply:
FWIW, you don't the semicolon when the call (e.g. `plt.legend`) is in a function, only when it's a direct call, in the last line of the cell.

Reply:
> FWIW, you don't the semicolon when the call (e.g. `plt.legend`) is in a function, only when it's a direct call, in the last line of the cell.

Yeah, I know. But that was kept for the sake of consistency.